src/extension.ts

```diff
@@ -374,12 +374,14 @@ function provideHover(document, position, token, syntax): Promise<vscode.Hover>
         } catch (err) {
             // Bad document
             resolve(null);
+            vscode.window.showInformationMessage(`Error: ${err}`);
             return;
         }
 
         // Not a k8s object.
         if (!obj.kind) {
             resolve(null);
+            vscode.window.showInformationMessage('Not a Kubernetes object!');
```

Comment:
Again, this is legit - if explain is on, the hover provider will fire on _all_ JSON and YAML documents.  We don't want to bug the user with "fool, you switched to a non-k8s document while explain was on"; we should just silently resolve as 'no hover tip'.

Reply:
Yep, sorry I got over aggressive with my rejecting :)